cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs

```diff
@@ -101,6 +103,14 @@ import           Data.Fixed (Fixed(MkFixed))
 import qualified Data.Yaml as Yaml
 import           Text.JSON.Canonical (parseCanonicalJSON, renderCanonicalJSON)
 
+import           Cardano.CLI.Shelley.Run.Genesis.ListMap (ListMap(..))
+import           Cardano.CLI.Shelley.Run.Genesis.LazyToJson (LazyToJson(lazyToJson))
+import           Cardano.CLI.Shelley.Run.Genesis.Types (toOutputTemplate)
+
+import qualified Cardano.CLI.Shelley.Run.Genesis.Types as OT
+import qualified Data.ByteString.Builder as B
```

Comment:
`Data.ByteString.Builder` should be with the other `Data.*` modules above

Reply:
Moved